OpenSim/Common/Test/testDataTable.cpp

```diff
@@ -701,5 +697,60 @@ int main() {
         std::cout << tableSVec << std::endl;
     }
 
+    {
```

Comment:
Perhaps add a comment to state what this block is testing (or a `std::cout` as has been done with the other blocks in this file)?

Reply:
Done.